### ISSUE TITLE
Functions using `readxl` may not function in R < 3.6.0

### ISSUE THREAD

**Issue by mhpob** [ocean-tracking-network/glatos#242]
TL;DR: consider bumping R version requirement to >= 3.6.0 to match `readxl`, or check functions using `readxl` to make sure they can still run on R v3.5.0.

Longer: `glatos` requires R v3.5.0. The following packages require R v3.6.0:

**Imports**
- `readxl`

**Suggests**
- `knitr`
- `testthat`

The packages in Suggests don't matter -- they're only there for testing and building vignettes. However, the functions using `readxl` should be checked to make sure they aren't leveraging R features only available in v3.6.0. Even so, the user will likely have to jump through some hoops since the dependency installation will probably break. Note that v3.5.0 was released in April 2024 and so is now 6+ years old.

See the test here for reference: https://github.com/ocean-tracking-network/glatos/actions/runs/10582439208/job/29322130768#step:5:169

Functions using `readxl`: https://github.com/search?q=repo%3Aocean-tracking-network%2Fglatos+readxl+path%3A*.r&type=code
- `prepare_tag_sheet`
- `prepare_deploy_sheet`
- `read_glatos_workbook`
- `read_vemco_tag_specs`

**Comment by chrisholbrook** [comment]
Browing https://github.com/tidyverse/readxl/blob/main/NEWS.md and commits around https://github.com/tidyverse/readxl/commit/f5bdfa35837fcc1793720959e43b755362732e4f, it's not clear what motivated requirement for R >= 3.6 or what was affected. But given the age, I'll make the change.

**Comment by chrisholbrook** [comment]
fixed in  757dc6dc4b815d7e1fc8c6f16428a2ada3b77c58
